client/src/FiltersPage.js

```diff
@@ -0,0 +1,181 @@
+import './App.css';
+import Login from './Login';
+import CardComponent from './CardComponent';
+import Filters from './Filters';
+import NavBar from './navbar';
+import React from 'react';
+import './Filter.css';
+
+export default function FiltersPage() {
+  ///////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////
+  // Map and List view mode buttons
+
+  const [mode, setMode] = React.useState(1);
+
+  function render() {
```

Comment:
don't call this `render` since that has a very specific meaning in React, use `renderCards` instead